plugins/system/deborphan

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
+#
+# Plugin to monitor the number of orphaned packages on the
+# system (using deborphan). Might work on section distib, who knows...
+#
+# Based on the debsecan plugin by Nicolas Bouthors. 2016-09-02
+#
+# Olivie Mehani <shtrom@ssji.net>
+#
+# Licence : GPLv2
+#
+#%# family=auto
+#%# capabilities=autoconf
+
+# Auto enable if we have deborphan only
+if [ "$1" = "autoconf" ] ; then
+  if [ -x /usr/bin/deborphan ]; then
```

Comment:
Please use `which deborphan >/dev/null` for a distribution/packaging-independent test.